### ISSUE TITLE
Coloring points by class on biplot

### ISSUE THREAD

**Issue by SamGG** [fbertran/plsRglm#4]
Dear authors,
Thanks for this interesting package. I am doing a logistic regression and I would to color the individuals on the biplot with a color for each class (2 in my case). Is it possible? The col argmument seems to be assigned to the two information datasets, not to points within each datasets.
Best,
Samuel

**Comment by fbertran** [comment]
You are right. This is a not directly possible. An exported classbiplot() helper was added to the package that keeps the usual biplot geometry but lets the individual layer be colored by class. It works with plsR and plsRglm fits, supports choosing components, optional legends, and point labels.

data(Cornell)
modpls <- plsR(Y ~ ., data = Cornell, nt = 2)
grp <- factor(Cornell$Y > median(Cornell$Y), labels = c("Low", "High"))
classbiplot(modpls, group = grp, col = c("firebrick3", "steelblue3"))
